src/main/java/org/hl7/davinci/priorauth/Endpoint.java

```diff
@@ -68,9 +67,9 @@ public static Response read(String resourceType, Map<String, Object> constraintM
                 Claim claim = (Claim) baseResource;
                 formattedData = requestType == RequestType.JSON ? App.DB.json(claim) : App.DB.xml(claim);
             } else if (resourceType == Database.CLAIM_RESPONSE) {
-                ClaimResponse claimResponse = (ClaimResponse) baseResource;
-                formattedData = requestType == RequestType.JSON ? App.DB.json(claimResponse)
-                        : App.DB.xml(claimResponse);
+                Bundle bundleResponse = (Bundle) baseResource;
```

Comment:
Maybe this can be rewritten since Bundle and ClaimResponse are handled the same way now. Perhaps we should make the claim come back as a Bundle as well so that they all work the same.

Reply:
Yea we can do that. The Claim endpoint is only for development purposes anyway so it would simplify the logic. I will update quickly

Reply:
PRIORAUTH-33 created to deal with this